docs/Operators.md

```diff
@@ -9325,6 +9325,10 @@ Other versions of this operator: <a href="Changelog.md#GreaterOrEqual-12">12</a>
   Given an `input` and a flow-field `grid`, computes the `output` using `input` values and pixel locations from `grid`.
   Currently, only spatial (4-D) inputs are supported. For `input` with shape (N, C, H, W) and `grid` with shape (N, H_out, W_out, 2),
   the `output` will have shape (N, C, H_out, W_out).
+
+  Input are pixels in 2-dimensional space with each pixel as a square whose value is at the square center.
```

Comment:
I think this should be explained better. I personally like what PR 4089 says. But as a compromise and a short summary for now, how about:
```
The tensor `Input` describes the value of a function (over a 2-dimensional space), or pixel, at integer coordinates.
The tensor `grid` describes the grid-positions where the function-value is to be computed using a specified
interpolation method (the mode) and a padding mode (for grid positions falling outside the dimensions of `input`)
```

Reply:
May be `contains` would be better than `describes` in above.

Reply:
So, something like
```
The tensor `Input` contains the pixel-values at integer-coordinates. The tensor `grid` describes the grid-positions
(using floating-values) where the pixel-value is to be computed using ... etc. (as above).
```